packages/rpc-graphql/package.json

```diff
@@ -73,8 +73,9 @@
         "@solana/build-scripts": "workspace:*",
         "@solana/eslint-config-solana": "^1.0.2",
         "@solana/keys": "workspace:*",
-        "@solana/rpc-core": "workspace:*",
-        "@solana/rpc-transport": "workspace:*",
+        "@solana/rpc": "workspace:*",
+        "@solana/rpc-subscriptions": "workspace:*",
```

Comment:
The subscriptions package shouldn't be needed either.

Reply:
Hmm not sure why but I thought it was imported somewhere in this package haha. Thanks for raising this, I'll remove the dependency.